src/tracker/engaged_time.js

```diff
@@ -85,9 +85,19 @@ limitations under the License.
         root.focused = false;
     });
 
+    var supportsPassive = false;
+    try {
+        addEventListener("test", null, Object.defineProperty({}, 'passive', {get: function () {
```

Comment:
@rbyers I'm having a hard time grokking this logic. I understand it's meant to determine whether the browser supports passive listeners, but I don't exactly see how it's doing that. Can you clarify?